src/lib/CRSF/CRSF.cpp

```diff
@@ -730,24 +741,24 @@ bool CRSF::UARTwdt()
             adjustMaxPacketSize();
 
             SerialOutFIFO.flush();
-#ifdef PLATFORM_ESP32
-            CRSF::Port.flush();
-            CRSF::Port.updateBaudRate(UARTrequestedBaud);
-#else
-            CRSF::Port.begin(UARTrequestedBaud);
-            #if defined(TARGET_TX_GHOST)
-            USART1->CR1 &= ~USART_CR1_UE;
-            USART1->CR3 |= USART_CR3_HDSEL;
-            USART1->CR2 |= USART_CR2_RXINV | USART_CR2_TXINV | USART_CR2_SWAP; //inverted/swapped
-            USART1->CR1 |= USART_CR1_UE;
+            #if defined(PLATFORM_ESP32) || defined(PLATFORM_ESP8266)
```

Comment:
Should be keep order similar to other places (8266 and then 32?)?
Or maybe even define and use generic `PLATFORM_ESP`?